libbeat/monitoring/opts.go

```diff
@@ -0,0 +1,46 @@
+package monitoring
+
+// Option type for passing additional options to NewRegistry.
+type Option func(options) options
+
+type options struct {
+	publishExpvar bool
+}
+
+var defaultOptions = options{
+	publishExpvar: false,
+}
+
+// PublishExpvar enables publishing all registered variables via expvar interface.
+// Note: expvar does not allow removal of any stats.
+func PublishExpvar(o options) options {
```

Comment:
I'm thinking if we should move this to `libbeat/monitoring/publisher` into its own namespace. Just in case we have more publishers for metrics in the future ;-)